README.md

```diff
@@ -97,7 +97,8 @@ There are multiple supported backends, which can be selected at runtime in the e
 | `/gpu/occa`              | CUDA OCCA kernels                                 |
 | `/omp/occa`              | OpenMP OCCA kernels                               |
 | `/ocl/occa`              | OpenCL OCCA kernels                               |
-| `/gpu/cuda`              | Pure CUDA kernels                                 |
+| `/gpu/cuda/ref`          | Pure CUDA kernels                                 |
```

Comment:
Could you please update these descriptions to briefly describe the difference between these two backends?